### PR TITLE
Add Incompatibles for Graphic Herbalism and Improved Argonians

### LINKED ISSUE #6: Add List of Incompatible Mods for Graphic Herbalism - MWSE Patches
While it doesn't (on its own) have an ESP, we can save people a bit of work by checking for it using its main patch:

They can't have both of the following, just one:

* `correctUV Ore Replacer_fixed.esp`
* `correctUV Ore Replacer_respawning.esp`

And the following ESPs would be incompatible with either if detected:

* `Graphic Herbalism Extra (Correct UV).esp`
* `correctUV Ore Replacer 1.0.esp`
* `Graphic Herbalism.esp`
* `Graphic Herbalism Extra.esp`
* `Graphic Herbalism TR.ESP`
* `Graphic Herbalism TR Extra.ESP`

### PR DESCRIPTION
**Pull request by Tyler799**
Hopefully that's right. :)

Fixes #6 and #7

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -118,6 +118,41 @@ plugins:
       - name: 'Patch for Purists - Floating Rocks.esm'
       - name: 'Patch for Purists - Expansion Implementation.esm'
       - name: 'Patch for Purists - Dialogue Inconsistencies.esm'
+      
+#Other Mods (default)
+  - name: 'Improved Argonians.esp'
+    group: *default
```
Comment (MacSplody) [inline masterlist.yaml]: You don't need to assign the `default` group, all plugins are included in it by default.

Comment (Tyler799) [reply masterlist.yaml]: Thanks!

**Comment by MacSplody** [comment]
@Tyler799 Thanks

**Comment by Tyler799** [comment]
Glad to help! :)

**Pull request merged**